docs/node-operators/index.mdx

```diff
@@ -53,7 +62,7 @@ A node is a machine running the Mina daemon. Different nodes fulfill different r
 
 3. SNARK Workers -  SNARK workers create [zk-SNARKs](https://minaprotocol.com/blog/what-are-zk-snarks) for each transaction. These zk-SNARKs are used to create recursive zk-SNARKs that prove the correctness of a block, and in turn, these zk-SNARKs are used to create recursive zk-SNARKs that prove the correctness of the network. These zk-SNARKs help provide the Mina Protocol with succinctness. 
 
-4. [Archive Nodes](https://docs.minaprotocol.com/node-operators/archive-node) - A regular mina daemon that is connected to a running archive process. The daemon regularly sends blockchain data to the archive process that stores it in a [PostgreSQL](https://www.postgresql.org/) database.
+4. [Archive Nodes](https://docs.minaprotocol.com/node-operators/archive-node) - A regular mina daemon that is connected to a running `mina-archive` process. The daemon regularly sends blockchain data to the archive process that stores it in a [PostgreSQL](https://www.postgresql.org/) database.
```

Comment:
```suggestion
4. [Archive Nodes](/node-operators/archive-node) - A regular mina daemon that is connected to a running `mina-archive` process. The daemon regularly sends blockchain data to the archive process that stores it in a [PostgreSQL](https://www.postgresql.org/) database.
```

Reply:
if you change yoru new directory name to singular `archive-node` this link will work

If you keep plural, you'll need to change all the links throughout docs and add redirect

I see a real scramble coming when we merge into main, eeeekkkkk

Reply:
yes it's good to merge frequently, but I did mention this when you insisted on this separate branch approach. It would be good to have this PR merged so we can see the state on the temp site generated by vercel. Then do small incremental fixes as required. This all works fine on my local build (and yes I use npm run build / npm run serve to test this.)

Reply:
best to get the "scramble" done before it goes live